dynamiqs/mesolve/mesolve.py

```diff
@@ -42,11 +43,14 @@ def mesolve(
     # === stepsize controller
     stepsize_controller, dt = _stepsize_controller(solver)
 
+    # === progress bar
+    bar, update_progressbar = make_progressbar(options.progress_bar, tsave[-1])
```

Comment:
We maybe need to specify `tsave[0]` too bc we solve from `t0=tsave[0]`? Ideally `t0` should be an option in `options`.